From 1e0368d74b445c55e094479c69298c7e55a42d42 Mon Sep 17 00:00:00 2001
From: Jeffery To <jeffery.to@gmail.com>
Date: Mon, 18 Feb 2019 16:23:45 +0800
Subject: [PATCH] python,python3: Fix calling default PyPackage/install

This fixes Package/*/install to call PyPackage/*/install correctly.
Previously, if a package used the default PyPackage/*/install, then it
would not called. (A custom-defined PyPackage/*/install would be called
with no issue.)

Signed-off-by: Jeffery To <jeffery.to@gmail.com>
---
 lang/python/python-package.mk  | 2 +-
 lang/python/python3-package.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/lang/python/python-package.mk b/lang/python/python-package.mk
index 855dd44..dc5d020 100644
--- a/lang/python/python-package.mk
+++ b/lang/python/python-package.mk
@@ -68,7 +68,7 @@ define PyPackage
   $(call shexport,PyPackage/$(1)/filespec)
 
   define Package/$(1)/install
-	$(call PyPackage/$(1)/install,$$(1))
+	$$(call PyPackage/$(1)/install,$$(1))
 	find $(PKG_INSTALL_DIR) -name "*\.exe" | xargs rm -f
 	SED="$(SED)" \
 	$(SHELL) $(python_mk_path)python-package-install.sh "2" \
diff --git a/lang/python/python3-package.mk b/lang/python/python3-package.mk
index 5a779c2..8bdac65 100644
--- a/lang/python/python3-package.mk
+++ b/lang/python/python3-package.mk
@@ -67,7 +67,7 @@ define Py3Package
   $(call shexport,Py3Package/$(1)/filespec)
 
   define Package/$(1)/install
-	$(call Py3Package/$(1)/install,$$(1))
+	$$(call Py3Package/$(1)/install,$$(1))
 	find $(PKG_INSTALL_DIR) -name "*\.exe" | xargs rm -f
 	SED="$(SED)" \
 	$(SHELL) $(python3_mk_path)python-package-install.sh "3" \
-- 
2.22.1

